steward/src/cellars/aave_v2_stablecoin.rs

```diff
@@ -2,37 +2,47 @@ use ethers::{abi::AbiEncode, prelude::H160};
 use steward_abi::aave_v2_stablecoin::*;
 use steward_proto::steward::aave_v2_stablecoin::Function::{self, *};
 
-pub fn get_encoded_call(function: Function) -> Vec<u8> {
+use super::log_cellar_call;
+
+const CELLAR_NAME: &str = "aave_v2_stablecoin";
+
+pub fn get_encoded_call(function: Function, cellar_id: String) -> Vec<u8> {
     match function {
         EnterStrategy(_) => {
+            log_cellar_call(CELLAR_NAME, "enterStrategy", cellar_id);
```

Comment:
Can you get the raw string (or similar) `enterStrategy` by inspecting the `EnterStrategyCall` struct you create here, so that you don't have to hardcode the string?